src/FSharp.Markdown/Markdown.fs

```diff
@@ -13,6 +13,8 @@ open System.Collections.Generic
 // Definition of the Markdown format
 // --------------------------------------------------------------------------------------
 
+type MarkdownRange = { Line : int }
```

Comment:
Does `Line` here mean just the line number? As @ljw1004 mentioned, it would make sense to track the total offset (in number of characters) from the beginning of the file.

As Markdown is fairly line-based, I guess we could keep line & column number for the position (which might be easier to implement on top of the current parser). Also, one thing to consider is whether we want just the _start_ of a span or whether we want _start and end_ (I think having both is better idea.)